cli/src/commands/fix.rs

```diff
@@ -63,9 +65,21 @@ use crate::ui::Ui;
 /// can potentially increase or decrease the number of conflict markers.
 ///
 /// The external tools must accept the current file content on standard input,
-/// and return the updated file content on standard output. A tool's output will
-/// not be used unless it exits with a successful exit code. Output on standard
-/// error will be passed through to the terminal.
+/// and return the updated file content on standard output. This is requirement
+/// enables the possibility to fix any commit. The working copy does not need
```

Comment:
nit: "commit" -> "revision" throughout because it looks like we consistently use the latter before this patch